index.html

```diff
@@ -2,18 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Project name:</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Review Practice</title>
     <link rel="stylesheet" href="css/style.css">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
-
   </head>
 
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-<!-- 
-    <div>
-      <h3>Hello World, we are software developers!</h3>
-    </div>  -->
```

Comment:
It is advisable not to leave comments in code. So please remove comments in your code,